.github/workflows/helm.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Helm
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+    branches:
+      - main
+      - release-*
+  push:
+    branches:
+      - main
+      - release-*
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+    - name: Get Golang version
+      id: vars
+      run: |
+        GOLANG_VERSION=$(./hack/golang-version.sh)
+        echo "GOLANG_VERSION=${GOLANG_VERSION##GOLANG_VERSION ?= }" >> $GITHUB_ENV
+    - name: Install Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ env.GOLANG_VERSION }}
+    - run: make test-helm
```

Comment:
Question -- I do not see a `test-helm` make target added to the top-level Makefile as part of this PR. Was that target already defined (even though it was unused)?

Reply:
This was added in a previous commit:
```
# We also add top-level targets for testing make targets:
.PHONY: $(TEST_TARGETS)
TESTS_FOLDERS := $(patsubst ./tests/%/,%,$(sort $(dir $(wildcard ./tests/*/))))
TEST_TARGETS := $(patsubst %,test-%,$(TESTS_FOLDERS))
$(TEST_TARGETS): test-%:
	make -f tests/$(*)/Makefile test
```

This automatically generates `test-*` targets in the same way that we generate the `cmd-*` targets.

Reply:
Got it!